database/factories/ActionlogFactory.php

```diff
@@ -38,7 +38,7 @@ public function assetCheckoutToUser()
     {
         return $this->state(function () {
             $target = User::inRandomOrder()->first();
-            $asset = Asset::RTD()->inRandomOrder()->first();
+            $asset = Asset::inRandomOrder()->RTD()->first();
```

Comment:
This seems superfluous? But I can't say that I mind too much.

Reply:
It was in the wrong order and was causing the seeder to fail.